### ISSUE TITLE
Per-picture Information Model

### ISSUE THREAD

**Issue by igorsiem** [igorsiem/photopres#3]
A means of associating various metadata - including freeform text - with each picture in a folder. Data to be stored in a ~~TOML~~ JSON file in the directory.

**Comment by igorsiem** [comment]
Implemented as described. Per-image data is recorded in a file named `.photopres.json` in each directory. Information structure is hierarchical and allows for future expansion.

The main branch for this feature is merged into the #2 branch, because it was developed in parallel with image viewing.
